dags/bqetl_adm_engagements_daily.py

```diff
@@ -44,7 +44,7 @@
         dataset_id="telemetry_derived",
         project_id="moz-fx-data-shared-prod",
         owner="akomar@mozilla.com",
-        email=["akomar@mozilla.com", "rharter@mozilla.com", "tbrooks@mozilla.com"],
+        email=["akomar@mozilla.com", "tbrooks@mozilla.com"],
```

Comment:
Is @teonbrooks still a relevant owner here in his PM role? cc @akkomar

Reply:
I would like to note that this dag should be considered retired after the switch from remote settings to contile for servicing the new tab sponsored shortcuts. We might need a separate conversation for this but for history and continuity of click data, we should figure out how to get this data to new collection mechanism. @rebecca-burwei, do you have a

Reply:
Is this `adm_engagements_daily` table as it currently exists no longer useful? Should I go ahead and remove it?

Reply:
just discussed internally. we can end this job. I would like to keep the table because it does contain historical data this is useful. we might need to trim the dates of it but I will open that up as a separate discussion topic